docs: Edge Cloud-init reference page (#1153)
spectrocloud/librarium: content/docs/04-clusters/03-edge/09-site-deployment/04-stage.md

6. Transfer the VMDK to a datastore in your VMware environment. Review the commands below and ensure you replace the placeholders with the respective values from your environment.

  ```shell
  export GOVC_URL=https://[IP address OR the DNS of vCenter] 
  export GOVC_USERNAME=[vcenter username]
  export GOVC_PASSWORD=[vcenter password]
  govc datastore.upload -ds=[datastore name] images/[your-installer-name].vmdk [folder in datastore]]/[your-installer-name].vmdk
  govc datastore.cp -ds=[datastore name] [folder in datastore]]/[your-installer-name].vmdk [folder in datastore]]/[your-installer-name]-uncompressed.vmdk
  ```

  If you are using test or development environments, you may need to enable the following option. This environment variable is not recommended for production environments.

  ```shell
  export GOVC_INSECURE=1
  ```

7. Create a VM from the VMDK by login into your vCenter console in the UI.

8. Navigate to the **Dataceter/Folder**, under the **VMs and Templates** section.

9. Start the **New Virtual** machine deployment wizard.

> 10. Choose a cluster from where you can access the datastore used for storing the VMDK. Choose the Datastore where VMDK is stored.

Author: ```suggestion
10. Choose a cluster that has access to the datastore used for storing the VMDK. Choose the Datastore where VMDK is stored.
```
